pkg/artifacts/artifact_storage_test.go

```diff
@@ -155,6 +237,15 @@ func TestInitializeArtifactStorageWithConfigMap(t *testing.T) {
 			if err != nil {
 				t.Fatalf("Somehow had error initializing artifact storage run out of fake client: %s", err)
 			}
+			if c.storagetype == "pvc" {
```

Comment:
could you include a comment here explaining what this block is for and why `CleanupArtifactStorage` is now being called in this test?

Reply:
Yup.